packages/component-base/src/a11y-announcer.d.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * @license
+ * Copyright (c) 2022 Vaadin Ltd.
+ * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+ */
+
+/**
+ * Cause a text string to be announced by screen readers.
+ */
+export function announce(text: string, options?: { mode: 'polite' | 'assertive'; timeout: number }): void;
```

Comment:
nit: `mode` and `timeout` should be optional so that `announce('hi', { mode: 'assertive' });` would not show an error
```suggestion
export function announce(text: string, options?: { mode?: 'polite' | 'assertive'; timeout?: number }): void;
```